code/_onclick/cyborg.dm

```diff
@@ -90,6 +98,35 @@
 		return
 	return
 
+//Gripper Handling
+//This is used when a gripper is used on anything. It does all the handling for it.
+//Lots of sanity checking because i don't want runtimes.
+
+/mob/living/silicon/robot/proc/GripperClickOn(var/atom/A, var/params, var/obj/item/weapon/gripper/G)
+	if(!gripper_sanity_check(G))
+		return
+	G.force_holder = G.wrapped.force
```

Comment:
This line and the next need to be placed inside the `if()` block below them, else `G.wrapped.force` could be set to 0 and left there if the `if()` fails.